index.html

```diff
@@ -1859,7 +1859,7 @@ <h2>Security Vocabulary Definitions</h2>
             it is a "pure" JSON pointer.
             Since this value is not a fragment identifier, it also does not
             need to URL-encode special characters.
-            The targeted element may or may not already exist at the specified location in the referenced data schema.
+            The targeted element may or may not already exist at the specified location in the referenced object or array schema.
```

Comment:
```suggestion
            The targeted element may or may not already exist at the specified location in the referenced object or array schema (consequently the mechanism is not applicable to simple types).
```

Does it make sense to be more specific?

Reply:
Why not :)